cumulusci/utils.py

```diff
@@ -404,37 +404,122 @@ def doc_task(task_name, task_config, project_config=None, org_config=None):
     from cumulusci.core.utils import import_global
 
     doc = []
-    doc.append("{}\n==========================================\n".format(task_name))
-    doc.append("**Description:** {}\n".format(task_config.description))
-    doc.append("**Class::** {}\n".format(task_config.class_path))
+    doc.append(f"**{task_name}**\n==========================================\n")
+    doc.append(f"**Description:** {task_config.description}\n")
+    doc.append(f"**Class:** {task_config.class_path}\n")
 
     task_class = import_global(task_config.class_path)
+
     if "task_docs" in task_class.__dict__:
         task_docs = textwrap.dedent(task_class.task_docs.strip("\n"))
-        doc.append(task_docs + "\n")
-    if task_class.task_options:
-        doc.append("Options:\n------------------------------------------\n")
-        defaults = task_config.options or {}
-        for name, option in list(task_class.task_options.items()):
-            default = defaults.get(name)
-            if default:
-                default = " **Default: {}**".format(default)
-            else:
-                default = ""
-            if option.get("required"):
-                doc.append(
-                    "* **{}** *(required)*: {}{}".format(
-                        name, option.get("description"), default
-                    )
-                )
-            else:
-                doc.append(
-                    "* **{}**: {}{}".format(name, option.get("description"), default)
-                )
+        doc.append(task_docs)
+
+    task_option_info = get_task_option_info(task_config, task_class)
+    doc.append("Command Syntax\n------------------------------------------\n")
+    command_syntax = get_command_syntax(task_name)
+    doc.append(command_syntax)
+
+    task_option_doc = create_task_options_doc(task_option_info)
+    if task_option_doc:
+        doc.append("Options\n------------------------------------------\n")
+        doc.extend(task_option_doc)
+
+    task_help_doc = create_task_help_doc(task_config)
+    if task_help_doc:
+        doc.append(
+            f"Help for task {task_name}\n------------------------------------------\n"
+        )
+        doc.extend(task_help_doc)
```

Comment:
These lines don't have coverage. You can probably just remove create_task_help_doc